gensim/models/keyedvectors.py

```diff
@@ -1943,14 +1943,19 @@ class FastTextKeyedVectors(WordEmbeddingsKeyedVectors):
         If True, uses the Facebook-compatible hash function instead of the
         Gensim backwards-compatible hash function.
 
+    Some important attributes:
```

Comment:
> The section doesn't actually show up in the docs, though.

yes (I guess reason in the custom template), but this is not important in the current case, because parameters + descriptions rendered correctly.

I guess just delete `Some ...` line and that's all

Reply:
OK, done.